resources/definitions/fdmprinter.def.json

```
@@ -2528,6 +2528,18 @@
                     "settable_per_extruder": false,
                     "settable_per_meshgroup": false
                 },
+                "material_max_mm3_per_sec":
+                {
+                    "label": "Max Filament Flow Rate",
+                    "description": "When non-zero, sets the maximum allowed filament flow rate (in mm^3/S). The filament flow rate of each printed line is calculated as layer height * line width * print speed. The flow rate limit is enforced by reducing the print speed. Note that these speed reductions are not taken into account when calculating the print time.",
```

Made the description less verbose, and more consistent with the other setting descriptions
```suggestion
                    "description": "When greater then zero, the maximum allowed flow rate is limited to Max Filament Flow Rate (in mm\u00b3/s). The flow rate limit is enforced by reducing the print speed.",
```